### ISSUE TITLE
would it make sense to have binary releases on github?

### ISSUE THREAD

**Issue by jhutar** [marckhouzam/custom-prometheus-exporter#2]
Hello. Would you accept a build script like:

https://github.com/oliver006/redis_exporter/blob/master/release-github-binaries.sh

I now anything about go, but I should be able to modify that script quite easily, because I think it would be handy to have pre-compiled binaries available.

**Comment by marckhouzam** [comment]
Thanks for the suggestion @jhutar !  I haven't had time recently for this project but I agree it would be nice to have a release.

In the mean time you can maybe use the docker image?
https://hub.docker.com/repository/docker/marckhouzam/custom-prometheus-exporter

**Comment by jhutar** [comment]
Hello. I do not have an immediate issue - we built the exporter and it just works. Would you be OK if I attempt to submit a PR with a script as above?

**Comment by marckhouzam** [comment]
Yes of course!  But I cannot promise when I'll have time to look at it, but I'll try.

**Comment by jhutar** [comment]
Sure, no problem. Thank you!
